File: src/main/java/no/nav/boot/conditionals/OnK8sCondition.java
package no.nav.boot.conditionals;

import org.springframework.core.type.AnnotatedTypeMetadata;

public class OnK8sCondition extends OnClusterCondition {

    @Override
    protected Cluster[] clusters(AnnotatedTypeMetadata md) {
        return Cluster.k8sClusters();
    }
}

File: src/main/kotlin/no/nav/boot/conditionals/OnK8sCondition.kt
package no.nav.boot.conditionals

import no.nav.boot.conditionals.Cluster.Companion.k8sClusters
import org.springframework.core.type.AnnotatedTypeMetadata

class OnK8sCondition : OnClusterCondition() {
    override fun clusters(md: AnnotatedTypeMetadata) = k8sClusters()
}